### ISSUE TITLE
installer/portable.sh doesn't work

### ISSUE THREAD

**Issue by comio** [filebot/plugins#11]
Hi @rednoah 

I'm trying to use portable.sh in order to download and install filebot automatically.
Furthermore the download fails creating a corrupted FileBot.tar.xz.gpg file (that contain the string "NOT FOUND").

Thanks in advance,

luigi

**Comment by rednoah** [comment]
Sorry, the Linux beta release has been discontinued for the time being. You can use the venerable `FileBot 4.7.9` for the time being.

If you're interested in *FileBot for Linux* then I'd appreciate support via Patreon:
https://www.patreon.com/filebot

**Comment by comio** [comment]
First time on patreon. I'm interested to support you (I already sponsored via paypal in the past). How it work? BTW, I able to use ant, javac, ... but I prefer to give you support for this tool if you can provide an aligned portable release.
Can you give more visibility to patreon link on your website?

**Comment by rednoah** [comment]
The majority of users are on Windows and Mac, so I don't plan to put it on the landing page. But it's in the Announcement section in each Linux related sub forum.

Unfortunately, most Linux users probably have some kind of automated setup and never check unless it stops working.

**Comment by comio** [comment]
I understand that you need to have a profitable way to give best product and support. Why cannot you add a link to patron page? You need to gain visibility and I don't understand your choice. If the majority of users use Win or Mac, you can catch also a lot of users from docker and *x world. Your product is a companion application for a lot of media servers (ie plex) that run on linux, nas or docker containers. You must give a chance. You must give a "special" benefit for paying users, like an already built and update release.

Why don't tell "Hey, I give support also to you, linux user, but you need to became a supporter/patreon to have a built release"

In my case, usually I use the forums only when I need. And a link in a nested/hidden page doesn't help to give you the required visibility.

In short: put a big button on homepage to your patreon page! ;)

See you in the universe.

ciao

luigi

**Comment by rednoah** [comment]
I have updated the `portable.sh` install script to print a message and advertise the Patreon page:
https://github.com/filebot/plugins/blob/master/installer/portable.sh

Please read it and let me know if you have any suggestions for improvement. I'm inclined to add the same massage to the `sysinfo` and `amc` scripts for which I can push automatic updates.

**Comment by lanrat** [comment]
The new `portable.sh` readme gets printed on every run when using filebot inside docker. Is it possible to mute the output of the readme or add a flag to disable the echo?

**Comment by rednoah** [comment]
What would be the point of that? The rest of the script doesn't work anyway, because the link doesn't exist, because FileBot for Linux builds don't exist.

**Comment by lanrat** [comment]
I'm not sure I understand your response.

Right now I use filebot inside docker, and log the output. The new readme in portable.sh is going into my logs and printing for every file it processes.

**Comment by rednoah** [comment]
I'm not sure I understand your response. This script is used for installing FileBot, which is currently not possible on Linux. If you already have an older version of `filebot` working, why would you call the install script?

**Comment by lanrat** [comment]
I'm not sure, I'm not calling it directly, but it is called every time inside docker.

**Comment by lanrat** [comment]
Here is an example of what I am talking about: https://asciinema.org/a/bJKe2yX1th5lxxTQMcJ1p5zN4

**Comment by rednoah** [comment]
My docker image does not use the `installer/portable.sh` script. What makes you think that the `installer/portable.sh` script has anything to do with your usage of the docker image?

-----

Are you perhaps referring to the amc script?

**Comment by lanrat** [comment]
Yes, that appears to be the case. Initially I could only find the readme in the portable.sh script.

Is there a way to silence it after the first run? I can create a new thread on the scripts repository if you would prefer. I did not mean to hijack this thread.

**Comment by rednoah** [comment]
I see. I think I added `#` to the beginning of each line so you can easily `grep -v` away those lines if you check the log.

**Comment by lanrat** [comment]
There does not appear to be a `#` in my case:
```

----------------------------------- <README> -----------------------------------

Dear Linux user, ?,

FileBot currently only officially supports Windows 10 and macOS because these platforms make it easy to sell FileBot via their respective app stores. Unfortunately, selling software on Ubuntu, Debian, Red Hat, SUSE, Synology NAS, QNAP NAS, etc is not as easy and not really worth the effort considering the small number of users willing to support the project. This Patreon is an experiment to see how many users are willing to financially contribute towards freely available donation-supported new releases on these platforms.

Please support FileBot for Linux with a pledge of $1 per release on Patreon:

=> https://www.patreon.com/filebot

If every other Linux user were to pledge a small amount of money to the FileBot project, then you could easily fund the project ten times over. Please financially support the software you use and rely on, be it FileBot or any other tool. It's just polite, and it helps the ecosystem.

Thank you,
The FileBot Team

---------------------------------- </README> -----------------------------------
```

**Comment by rednoah** [comment]
I see. You can use `dev:amc` to run the latest code directly from GitHub.

**Comment by lanrat** [comment]
Thanks! `dev:amc` added the `#` at the beginning of each line.
